config/cars/mods/urd/urd_t5_bayro_2018.ini

```diff
@@ -0,0 +1,96 @@
+[INCLUDE]
+INCLUDE=common/no_popup_lights.ini
+
+[LIGHTING]
+INTERIOR_FAKE_UPPER_SHADOW_HEIGHT=0.03
+INTERIOR_FAKE_UPPER_SHADOW_FADE=0.05
+EMISSIVE_HEADLIGHTS_MULT=3
+EMISSIVE_PARKINGLIGHTS_MULT=2
+EMISSIVE_BRAKELIGHTS_MULT=4
```

Comment:
Values are unreasonably high, re-test using several different post-process filters such as Sol to ensure that the visuals aren't broken for other users as BLOOM_LUMINANCE and other glare-related parameters may dim or brighten the emissives.